enricher/api/src/main/java/io/fabric8/maven/enricher/api/BaseEnricher.java

```diff
@@ -228,4 +190,16 @@ protected void setProcessingInstruction(String key, List<String> containerNames)
         processingInstructionsMap.put(key, String.join(",", containerNames));
         enricherContext.setProcessingInstructions(processingInstructionsMap);
     }
+
+    protected Boolean getBooleanValueFromConfig(String propertyName, Boolean defaultValue) {
```

Comment:
@rohanKanojia why not let the name be `getValueFromConfig` and put function overloading to use ? @lordofthejars thoughts?

Reply:
Nice catch! I would update this PR